package.json

```diff
@@ -1,8 +1,8 @@
 {
-	"name": "ACT-RULES-CG",
+	"name": "ACT-Rules-Community",
```

Comment:
I don't think NPM allows capitalisation in package names at all, does it? Maybe just pull from package.author, rather than using package.name.